### PR TITLE
Use shorthand property name in module.exports

### PR DESCRIPTION
**Pull request by umaar**

### CODE REVIEW

**Comment by samccone** [comment]
:+1: nice

File: helpers/emulation.js [hunk 1]
```diff
@@ -92,8 +92,4 @@ function enableNetworkThrottling(driver) {
   return driver.sendCommand('Network.emulateNetworkConditions', TYPICAL_MOBILE_THROTTLING_METRICS);
 }
 
-module.exports = {
-  enableNexus5X: enableNexus5X,
-  enableNetworkThrottling: enableNetworkThrottling,
-  disableCache: disableCache
-};
+module.exports = {enableNexus5X, enableNetworkThrottling, disableCache};
```
Comment (brendankenny) [inline helpers/emulation.js]: not sure how others feel, but I actually prefer each item on its own line, for easy reading

Comment (samccone) [inline helpers/emulation.js]: :+1: to multiline

Comment (umaar) [inline helpers/emulation.js]: Done! Thanks for the feedback.

**Comment by brendankenny** [comment]
:boom:

**Comment by brendankenny** [comment]
Thanks!

**Pull request merged**